<launch>
    <node pkg="drive_recorder" name="drive_recorder" type="drive_recorder_node" output="screen">
        <param name="before_time" value="600"/>
        <param name="after_time" value="60"/>
        <param name="bag_period" value="60"/>
        <param name="poling_interval" value="2"/>
        <param name="log_dir" value="$(env HOME)/.ros/log/latest"/>
        <param name="log_out" value="$(env HOME)/.ros/log/backup"/>
    </node>
    <node pkg="drive_recorder" name="drive_recorder_test" type="drive_recorder_test" output="screen"/>
</launch>

